src/components/submission/show.vue

```diff
@@ -11,10 +11,7 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <div>
-    <page-back v-show="submission.dataExists" :to="formPath('submissions')">
-      <template #title>{{ $t('back.title') }}</template>
-      <template #back>{{ $t('back.back') }}</template>
-    </page-back>
+    <breadcrumbs :links="breadcrumbLinks"/>
     <page-head v-show="submission.dataExists">
       <template #title>{{ submission.dataExists ? submission.instanceNameOrId : '' }}</template>
     </page-head>
```

Comment:
Use full dataExists here:

```suggestion
    <page-head v-show="dataExists">
      <template #title>{{ submission.dataExists ? submission.instanceNameOrId : '' }}</template>
    </page-head>
```